indico/htdocs/sass/modules/_sessions.scss

```diff
@@ -0,0 +1,64 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+@import 'base';
+@import 'base/animation';
+@import 'compass';
+@import 'partials/icons';
+
+.sessions {
+    thead {
+        th {
+            &:not(.sorter-false) {
+                cursor: pointer;
+            }
+
+            &:before {
+                @include rotate-transform(0.3s);
+                display: inline-block;
+            }
+
+            &.header-sort-desc {
+                @include icon-before('icon-arrow-down');
+            }
+
+            &.header-sort-asc {
+                @include icon-before('icon-arrow-down');
+
+                &:before {
+                    @include rotate(-180deg);
+                }
+            }
+        }
+    }
+
+    td {
+        .text-box {
+            width: 8px;
+            height: 8px;
+            margin: auto;
+        }
+
+        .palette-color {
+            cursor: pointer;
```

Comment:
I forgot that I had moved it to another file and I simply didn't remove it from here.